libraries/index.html

```diff
@@ -0,0 +1,132 @@
+      on GitHub.
+    <div class="release-section">
+      <div id="adafruit-circuitpython-bundle-4.x-mpy">
+        <h3>Bundle Version 4.x</h3>
+        <p>
+          This bundle is built for use with CircuitPython 4.x.x. If you are using
+          CircuitPython 4, please download this bundle.
+        </p>
+      </div>
+      <div id="adafruit-circuitpython-bundle-3.x-mpy">
+        <h3>Bundle Version 3.x</h3>
+        <p>
+          This bundle is built for use with CircuitPython 3.x.x. If you are using
+          CircuitPython 3, please download this bundle.
+        </p>
+      </div>
+      <div id="adafruit-circuitpython-bundle-py">
+        <h3>Bundle Version py</h3>
+        <p>
+          This bundle is the uncompiled source code for every library. It is not
+          intended for general use! It is only recommended if you need to edit
+          a library file. This bundle works with all versions of CircuitPython.
+        </p>
+      </div>
+      <div id="adafruit-circuitpython-bundle-examples">
+        <h3>Bundle Examples</h3>
+        <p>
+          Every CircuitPython library has examples to go along with it. This bundle
+          contains every library example. This bundle will have examples that
+          require the latest version of CircuitPython, and therefore contains files
+          that may not work with all versions.
+        </p>
+      </div>
+    </div>
+    <h2>Contributing</h2>
+    <p>
+      If you'd like to contribute to the CircuitPython project, the CircuitPython
+      libraries are a great way to begin. Everyone is welcome to contribute!
+      Regardless of your experience level, we have many resources to get you
+      started. Check out the Contributing page for detailed information!
+    </p>
+    <p>
+      Find out more about how you can
+      <a href="{{ "/libraries/contributing" | relative_url  }}" title="Contributing Page">contribute to CircuitPython libraries</a>.
+    </p>
+    <h2>Library Activity for Week of  {{ site.data.libraries.updated_at | date: "%a, %b %d, %Y" }}</h2>
+    <p>
+      Thank you to all of our contributors!
+    </p>
+    <p>
+      This week there were [x] pull requests merged:
+      <ul>
+        <li>
+          Authors:
+          {% for contributor in site.data.libraries.contributors %}
+            <a href="https://github.com/{{contributor}}" title="Contributor Name">{{contributor}}</a>{% if forloop.last != true %}, {% endif %}
+          {% endfor %}
+        </li>
+        <li>
+          Reviewers:
+          {% for reviewer in site.data.libraries.reviewers %}
+            <a href="https://github.com/{{reviewer}}" title="Contributor Name">{{reviewer}}</a>{% if forloop.last != true %}, {% endif %}
+          {% endfor %}
+        </li>
+      </ul>
+    </p>
+    <p>
+      These are the new and updated libraries from the past week.
```

Comment:
Apologies for forgetting! I meant to have you take this line out. It looks odd tucked in there. The title "Library Activity etc" followed by the headers "New Libraries" and "Updated Libraries" is clear without it.